apps/admin/backend/src/store.ts

```diff
@@ -1250,6 +1258,37 @@ export class Store {
     return sheetNumber ? { type: 'hmpb', sheetNumber } : { type: 'bmd' };
   }
 
+  private parseVotesWithAdjudications({
+    votesString,
+    adjudicationsString,
+  }: {
+    votesString: string;
+    adjudicationsString: string | null;
+  }): Tabulation.Votes {
+    const votes = JSON.parse(votesString) as Tabulation.Votes;
+    if (!adjudicationsString) return votes;
+
+    const adjudications = JSON.parse(
+      adjudicationsString
+    ) as CastVoteRecordVoteAdjudication[];
+
+    for (const adjudication of adjudications) {
+      const currentContestVotes =
+        votes[adjudication.contestId] ?? /* c8 ignore next 1 */ [];
+      if (adjudication.isVote) {
+        votes[adjudication.contestId] = [
+          ...currentContestVotes,
+          adjudication.optionId,
+        ];
```

Comment:
It could only contain the adjudication `optionId` if either there was a redundant adjudication or a duplicate adjudication, and the logic & testing in `adjudication.ts` covers that. I don't want anyone to try to edit the adjudications table other than using those methods, but not sure how I could enforce that pattern.

Reply:
Re: order - no, not assuming a particular order.